packages/client/bin/cli.ts

```diff
@@ -213,6 +213,22 @@ async function run() {
       input: process.stdin,
       output: process.stdout,
     })
+
+    // Hide key input
+    ;(rl as any).input.on('keypress', function () {
+      // get the number of characters entered so far:
+      const len = (rl as any).line.length
+      // move cursor back to the beginning of the input:
+      readline.moveCursor((rl as any).output, -len, 0)
+      // clear everything to the right of the cursor:
+      readline.clearLine((rl as any).output, 1)
+      // replace the original input with asterisks:
+      for (let i = 0; i < len; i++) {
+        // eslint-disable-next-line no-extra-semi
+        ;(rl as any).output.write('*')
+      }
+    })
```

Comment:
This is a bit hacky, but due to time constraints I can't come up with a more clean solution and I would really want to have this in and not expose the private key on the screen. Feel free to directly improve yourself but please don't make this a blocker.